### ISSUE TITLE
Allow option to control mouseover and/or clickAction in treeNetwork

### ISSUE THREAD

**Issue by steadyfish** [christophergandrud/networkD3#198]
Currently `forceNetwork` function allows users to specify `clickAction` but `radialNetwork` and `diagonalNetwork` do not allow to control hover or click action. 

A particular usecase would be something similar to [this](http://stackoverflow.com/questions/36895960/linking-a-node-in-networkd3-to-a-website-using-clickaction-null), but for a `radialNetwork`.

Not sure if this would be covered under #197.

**Comment by cjyetman** [comment]
Related to #186 and #197

**Comment by cjyetman** [comment]
Also relevant to #171 since all the hierarchical functions will likely be merged into one `treeNetwork` function. I have a prototype for the new `treeNetwork` function in dev, and by default it is collapsible on click... I think that will always be on by default, so I'm not sure how/if it would be able to offer custom click behavior.

**Comment by cjyetman** [comment]
'mouseover' and 'mouseout' parameters are definitely in the dev version of `treeNetwork` I'm working on... if it ever sees the light of day

**Comment by cjyetman** [comment]
this will be possible with treeNetwork once PR #200 is merged. closing
